### DISCUSSION TITLE
A way to auto disable pane sync when going full

### DISCUSSION THREAD

**Discussion by yerlaser** [zellij-org/zellij#2826] [Ideas]
One issue I found with pane sync is that sometimes I want to maximize one pane which effectively hides the other panes.
However, if sync was active it remains so.
The problem happens that I'd continue to run commands in the active pane and often forget that the same commands execute on the others without me seeing it.
It can have catastrophic effect.

**Comment by Lockszmith-GH** [comment]
Looking at the documentation I see the action bound to the keyboard is `ToggleActiveSyncTab`.

* [CLI](https://zellij.dev/documentation/cli-actions.html?highlight=sync#toggle-active-sync-tab)
* [Plugin-API](https://zellij.dev/documentation/plugin-api-commands.html?highlight=sync#toggle_active_tab_sync)
* [KeyBinds](https://zellij.dev/documentation/keybindings-possible-actions.html?highlight=sync#toggleactivesynctab)

There is Explicit Off command.

There way I see it, you have a very solid 2 part feature request here:
1. Add an explicit on/off action to control Sync.
2. Make the default key-binding for FullScreen be: *SyncOff* followed by *ToggleFocusFullscreen*
